fixtures/base/10_runtimes.neon

```diff
@@ -15,6 +20,11 @@ App\Model\Entity\RuntimeEnvironment:
             - "[pas]"
             - "GNU/Linux"
             - "Pascal compiled with FreePascal"
+            - '''
+              - name: "source_files"
+                type: "file[]"
```

Comment:
This should accept a single file (as long as we don't want to implement building complex pascal apps)

Reply:
Will be repaired in near future, together with appropriate box.

Reply:
OK